instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integrationtests/profiling/MemoryProfileForLogs.kt

```diff
@@ -38,6 +38,6 @@ internal class MemoryProfileForLogs {
                     )
                 }
             }
-        }, threshold = 400)
+        }, threshold = 800)
```

Comment:
Same as above

Reply:
Those are somehow manually determined based on try - outs. Idea was to find a limit for which the tests will pass and from now on whenever we add things to our Log models to try to stay under that limit. Will add these comments there.